nixup/modules/nixup/nixup-resources.sh

```diff
@@ -0,0 +1,124 @@
+#! @shell@
+
+rmEmptyDirectories () {
+	BASE=$1
+	DIR=$2
+	if [ "$DIR" != "." -a -d "$BASE/$DIR" ]; then
+		if [ ! "$(@coreutils@/bin/ls -A $BASE/$DIR)" ]; then
+			@coreutils@/bin/rm -rf $BASE/$DIR
```

Comment:
This can instead be done with `rmdir` without the `if`, since that will error out if the dir is missing or not empty.